src/main/java/HookKiller/server/board/controller/ArticleController.java

```diff
@@ -0,0 +1,67 @@
+package HookKiller.server.board.controller;
+
+import HookKiller.server.board.dto.ArticleRequestDto;
+import HookKiller.server.board.dto.PostArticleRequestDto;
+import HookKiller.server.board.service.ArticleContentService;
+import HookKiller.server.board.service.ArticleService;
+import HookKiller.server.board.type.BoardType;
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequestMapping("/articles")
+@RequiredArgsConstructor
+public class ArticleController {
+
+  private final ArticleService articleService;
+  private final ArticleContentService articleContentService;
+
+  /**
+   * 게시글 조회
+   */
+  @GetMapping("/{boardId}")
+  public List<ArticleRequestDto> getArticleList(@PathVariable Long boardId, HttpServletRequest request) {
+    BoardType language = BoardType.valueOf(request.getHeader("language"));
+    return articleService.getArticleList(boardId, language);
+  }
+
+  /**
+   * 게시글 등록
+   */
+  @PostMapping
+  public ResponseEntity<String> createArticle(@RequestBody PostArticleRequestDto requestDto) {
+    articleContentService.createContent(
+            requestDto, articleService.createArticle(requestDto)
+    );
+    return ResponseEntity.ok("Article Create Success");
+  }
+
+  /**
+   * 게시물 수정
+   */
+  @PostMapping("/{articleId}")
```

Comment:
게시물 수정이니 아래처럼 변경하고, articleId는 PathVariable이 아닌, RequestDto에 같이 받아서 처리하는건 어떨까요?
```suggestion
  @PostMapping
```